### PR TITLE
fix(SubdomainInput) fix subdomain input spacing

### PR DESCRIPTION
**Pull request by sergiuelisei**
https://etvas.atlassian.net/browse/ETV-3497

### CODE REVIEW

File: src/SubdomainInput/SubdomainInput.jsx [hunk 1]
```diff
@@ -46,6 +54,35 @@ export const SubdomainInput = forwardRef(
...
   ) => {
     const inputRef = useRef()
+    const totalWidth = useRef()
+    const [maxInputWidth, setMaxInputWidth] = useState()
+
+    const prefixWidth = useMemo(
+      () =>
+        getTextWidth(
+          prefix,
+          `${theme.fontWeights.lighter} ${theme.fontSizes[2]} ${theme.fonts.primary}`
```
Comment (ohnu) [inline src/SubdomainInput/SubdomainInput.jsx:64]: what do you think if we extract `${theme.fontWeights.lighter} ${theme.fontSizes[2]} ${theme.fonts.primary}` to a method ?

File: src/SubdomainInput/SubdomainInput.jsx [hunk 2]
```diff
@@ -134,13 +168,13 @@ export const SubdomainInput = forwardRef(
               readOnly={readOnly}
               ref={ref || inputRef}
               required={required}
-              width={inputWidth}
+              width={inputWidth <= maxInputWidth ? inputWidth : maxInputWidth}
```
Comment (ohnu) [inline src/SubdomainInput/SubdomainInput.jsx:171]: use Math.min function ?

File: src/SubdomainInput/SubdomainInput.jsx (hunk 1 above)
Comment (sergiuelisei) [reply src/SubdomainInput/SubdomainInput.jsx:65]: I think it's a good idea 👍🏻

File: src/SubdomainInput/SubdomainInput.jsx (hunk 2 above)
Comment (sergiuelisei) [reply src/SubdomainInput/SubdomainInput.jsx]: cb19b73

**Review by ohnu — APPROVED** [review]

**Review by alsofronie — APPROVED** [review]

**Pull request merged**